docs/ibc/upgrades/developer-guide.md

```diff
@@ -0,0 +1,40 @@
+# IBC Client Developer Guide to Upgrades
```

Comment:
```suggestion
# IBC Client Upgrade Guide
```

Reply:
This document is specifically for developers who are trying to implement an IBC client. It is not relevant for end users. Is there a better title you had in mind that makes it clear this is for developers?

Reply:
no, it's just that client developer sounds like wallets, exchanges, etc. Maybe light client devs?